cax/config.py

```diff
@@ -279,3 +289,43 @@ def adjust_permission_base_dir(base_dir, destination):
 
       subprocess.Popen( ["setfacl", "-R", "-M", "/cfs/klemming/projects/xenon/misc/basic", base_dir],
                         stdout=subprocess.PIPE )
+
+
+def get_adler32( fname ):
```

Comment:
Should this be moved to cax/tasks/checksum.py? (And where is this being used actually?)